src/card.tsx

```diff
@@ -1275,7 +1279,10 @@ export const PosterCard = React.forwardRef<HTMLDivElement, PosterCardProps>(
                     >
                         {isTouchable && <div className={styles.touchableCardOverlayMedia} />}
```

Comment:
When the card has an image or video defined should use the `touchableCardOverlayMedia` style, but when the card has no media defined or a custom color defined, depending on if it's inverse or not should use `touchableCardOverlayInverse` / `touchableCardOverlay`

<img width="692" alt="Screenshot 2023-08-10 at 09 02 18" src="https://github.com/Telefonica/mistica-web/assets/44420072/0b0a44c9-9b31-4a39-a22d-9ebc80f625f9">

Reply:
I did not understand your comment. Could you explain it better?

Reply:
Of course, there are three cases:

* **The card has an image or video defined:** The overlay that handle the `:hover` and `pressed` styles should use `touchableCardOverlayMedia` that makes the layer to have a black with 25% opacity when hovered and 35% when pressed, that works in most images since we don't know how dark/light they are.
* **The card has no image or video defined and is inverse={false}:** The overlay that handle the `:hover` and `pressed` styles should use `touchableCardOverlay`. Since the background will be white (the default) or a light color, the styles for the overlay are more subtle.
* **The card has no image or video defined and is inverse={true}:** The overlay that handle the `:hover` and `pressed` styles should use `touchableCardOverlayInverse`. Since the background will be brand (the default) or a dark color, the styles for the overlay should be a little bit stronger in order to be visible.

The same applies to the top actions container (image posted in the previous comment), depending of the case the container will have different default an interaction states.

* **The card has an image or video defined:** The actions container by default is `inverse `at 70%, and we change the opacity for the :hover (90%) and `pressed` (100%) states.
* **The card has no image or video defined and is inverse={false}:** The actions container by default is transparent and uses the tokens `backgroundContainerHover` & `backgroundContainerPressed` for the interaction states.
* **The card has no image or video defined and is inverse={true}:** The actions container by default is transparent and uses the tokens `backgroundContainerBrandHover` & `backgroundContainerBrandPressed` for the interaction states